### PR TITLE
New questions to GitHub Actions

### PR DESCRIPTION
**Pull request by dostonhamrakulov**
## PR Type

- [x] Adding new question(s)
> **Warning**: We do not support the inclusion of questions directly copied from official GitHub certification exams. Please only submit original questions and content that you have created.
- [ ] Other content changes (updating questions, answers, explanations or study resources)
- [ ] Code changes (non-content)
- [ ] Documentation changes
- [ ] Other

### CODE REVIEW

File: content/questions/actions/question-102.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+question: "Select the context property to access information about the event that triggered a workflow run?"
```
Comment (FidelusAleksander) [inline content/questions/actions/question-102.md:2]: seems more clear

```suggestion
question: "Which context holds information about the event that triggered a workflow run?"
```

**Review by FidelusAleksander — CHANGES_REQUESTED** [review]
Hi @dostonhamrakulov - just a small fix needed and we're good :+1:

File: content/questions/actions/question-102.md (hunk 1 above)
Comment (dostonhamrakulov) [reply content/questions/actions/question-102.md]: updated

File: content/questions/actions/question-102.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+question: "Which context holds information about the event that triggered a workflow run"
```
Comment (FidelusAleksander) [inline content/questions/actions/question-102.md:2]:
```suggestion
question: "Which context holds information about the event that triggered a workflow run?"
```

**Review by FidelusAleksander — APPROVED** [review]
Thanks for contributing again @dostonhamrakulov  :rocket:  !

**Pull request merged**